### ISSUE TITLE
Use of send.bind(send, 'METHOD') prevents tree shaking with Rollup

### ISSUE THREAD

**Issue by rdmurphy** [lukeed/httpie#34]
I'm working with the `next` v2 branch! This is so minor because it's like... four lines. 😅

I'm working on a library that bundles in `httpie` instead of importing it (because it's so small, so why not? 🙌 ) and I noticed that even though I was only importing `post` the [other four helpers](https://github.com/lukeed/httpie/blob/next/src/node.js#L72-L76) were also there. I guess Rollup is being extra careful with `bind` and includes them in case of any side effects?

When I converted it to something like this (not necessarily recommending it):

```js
export const get = (...args) => send('GET', ...args);
export const post = (...args) => send('POST', ...args);
export const patch = (...args) => send('PATCH', ...args);
export const del = (...args) => send('DELETE', ...args);
export const put = (...args) => send('PUT', ...args);
```

Then it'd tree shake just fine.

**Comment by lukeed** [comment]
Hey, cool!

I actually let Rollup know about this a while ago (https://github.com/rollup/rollup/issues/2566) and it was planned/on the roadmap supposedly.

So I'm actually keen on leaving it how it is because, as you said, it's hardly anything at all, and any production/Terser build will purge them.

...

Actually, reminds me to try playing with that `/*PURE*/` hint. I think Rollup may listen to that if done correctly (from httpie source)?

**Comment by rdmurphy** [comment]
I was mucking around directly in the `node_modules` copy of `httpie` to see what's up so I dropped that in and can confirm that `/*#__PURE__*/` **does** work. 🎉 This dropped the unused ones as expected (in Rollup 2.33.3):

```js
export const get = /*#__PURE__*/ send.bind(send, 'GET');
export const post = /*#__PURE__*/ send.bind(send, 'POST');
export const patch = /*#__PURE__*/ send.bind(send, 'PATCH');
export const del = /*#__PURE__*/ send.bind(send, 'DELETE');
export const put = /*#__PURE__*/ send.bind(send, 'PUT');
```

**Comment by lukeed** [comment]
Oo nice! Feel free to send PRs if you've the time. Otherwise I'll set a reminder for Monday (no laptop this weekend)

**Comment by rdmurphy** [comment]
(Sounds amazing! 🙏 )

And sure! I'll try to throw one in. Against `master` or `next` (or both)?

**Comment by lukeed** [comment]
Whichever you prefer/are using. I'll comb thru Monday to fill in anything missing and publish a patch

**Comment by lukeed** [comment]
I think I'm going to leave `1.x` branch alone. I _should_ get my act together and promote `httpie@next` to stable before year end. If you think it's necessary before I get to that, please leave a shout :)

Closed by your #35 & released in `v2.0.0-next.12` – thanks!
